src/renderer/views/Playlist/Playlist.js

```diff
@@ -15,6 +16,19 @@ import {
 import { extractNumberFromString, setPublishedTimestampsInvidious, showToast } from '../../helpers/utils'
 import { invidiousGetPlaylistInfo, youtubeImageUrlToInvidious } from '../../helpers/api/invidious'
 
+const SORT_BY_VALUES = {
+  // TODO: store video.published for user playlists
+  // DatePublishedNewest: 'date_published_newest',
+  // DatePublishedOldest: 'date_published_oldest',
```

Comment:
If you want to implement this, you'll have to keep in mind that shorts don't have a published date on the shorts channel tab or in shorts only playlists (not returned in youtube's response, thats's why we always use RSS on the shorts tab on the subscriptions page), so if someone were to add a video to a playlist from there, it would make it impossible to sort by published. For the watch history we just don't store the published date if it isn't available, but we also don't have any sorting functionality that would depend on it.

Reply:
Thanks for this context absidue. Just removedd the code for these.